google_health/delphi_google_health/run.py

```diff
@@ -95,12 +102,20 @@ def run_module():
     for signal in signal_names:
         is_smoothed = signal.endswith(SMOOTHED)
         for geo_res, df in df_by_geo_res.items():
-            create_export_csv(format_for_export(df, is_smoothed),
+            exported_csv_dates = create_export_csv(
+                              format_for_export(df, is_smoothed),
```

Comment:
I don't like the alignment here; shouldn't it just be one tab level under the "c" in "create_export_csv"?

Reply:
Done